rsocket-core/src/main/java/io/rsocket/stat/NoopStats.java

```diff
@@ -0,0 +1,65 @@
+package io.rsocket.stat;
+
+import io.rsocket.DuplexConnection;
+import io.rsocket.FrameType;
+
+/**
+ * Created by geleri on 5/25/17.
+ */
+public class NoopStats implements RSocketStats{
```

Comment:
This is a useful base class, maybe call this BaseStats and allow people to extend this